mne/epochs.py

```diff
@@ -272,9 +272,10 @@ def __init__(self, info, data, events, event_id, tmin, tmax,
             self.preload = False
             self._data = None
         else:
+            index_offset = 1 if data.shape[-1] > 1 else 0
             assert decim == 1
             if data.ndim != 3 or data.shape[2] != \
-                    round((tmax - tmin) * self.info['sfreq']) + 1:
+                    round((tmax - tmin) * self.info['sfreq']) + index_offset:
```

Comment:
Sorry, I still don't get it, you'll have to be a bit more explicit. What will be a small float value? `tmax-tmin` here?

I'm trying to get a handle on what failing values of `tmin`, `tmax`, and `data.shape` are at this point. It might be easiest if you add some tests that fail on `master` but pass here. I actually suspect the change here is necessitated by the change below `tmax = (data.shape[2] - index_offset) / info['sfreq'] + tmin` where `index_offset` has been changed to be zero instead of 1, as it was before. I think there might be a more principled solution to this...